backend/config.ts

```diff
@@ -19,6 +19,7 @@ export const app = {
   esweEnabled: process.env.ESWE_ENABLED === 'true',
   neurodiversityEnabledUsernames: process.env.NEURODIVERSITY_ENABLED_USERNAMES,
   disableRequestLimiter: process.env.DISABLE_REQUEST_LIMITER ? process.env.DISABLE_REQUEST_LIMITER === 'true' : false,
+  caseNotesBehaviourPrompts: process.env.CASE_NOTES_BEHAVIOUR_PROMPTS === 'true',
```

Comment:
feature flag because this feature is not meant to go live before a particular date